mirage/fixtures/crate-owner-invitations.js

```diff
@@ -0,0 +1,102 @@
+export default [
+  {
+    inviteeId: 1,
+    inviterId: 2,
+    crateId: 'mock-dev-deps',
+  },
+  {
+    inviteeId: 1,
+    inviterId: 2,
+    crateId: 'postgres',
+  },
+  {
+    inviteeId: 1,
+    inviterId: 2,
+    crateId: 'nom',
+  },
+  {
+    inviteeId: 1,
+    inviterId: 2,
+    crateId: 'rusted_cypher',
+  },
+  {
+    inviteeId: 1,
+    inviterId: 2,
+    crateId: 'nanomsg',
```

Comment:
could you try to use mirage factories instead? we're trying to move away from hard-coded fixtures :)